src/templates/contacts.html.j2

```diff
@@ -0,0 +1,64 @@
+{% extends "base.html.j2" %}
+{% block title -%}
+OSG Contacts
+{%- endblock %}
+{% block content %}
+{% if not authorized %}
+<div class="alert alert-info" role="alert">
+  To see contact information, load your grid certificate into your browser.
+</div>
+{% endif %}
+<div class="alert alert-info" role="alert">
+  Profile information is provided by the user and may be out of date.
+</div>
+<div class="container">
+  {%- for user in users %}
```

Comment:
I would prefer each user be in their own "div".  Then, you can do things like [bootstrap cards](https://getbootstrap.com/docs/4.1/components/card/) (kinda like contact cards)